docs/v1/P4Runtime-Spec.mdk

```diff
@@ -1957,10 +1957,27 @@ case (P4~16~ `bit<W>` type).
 Just like its P4Info counterpart - `P4DataTypeSpec` -, `P4Data` uses a Protobuf
 `oneof` to represent all possible values.
 
-The order of `members` in `P4StructLike`, the order of `bitstrings` in
-`P4Header`, and the order of `entries` in `P4HeaderStack` and
-`P4HeaderUnionStack` must match the order in the corresponding p4info.proto type
-specification and hence the order in the corresponding P4~16~ type declaration.
+We define a canonical representation for `P4Data` messages - therefore
+guaranteeing read-write symmetry - by introducing the following requirements:
+
+ * The order of `members` in `P4StructLike`, the order of `bitstrings` in
+   `P4Header`, and the order of `entries` in `P4HeaderStack` and
+   `P4HeaderUnionStack` must match the order in the corresponding p4info.proto
+   type specification and hence the order in the corresponding P4~16~ type
+   declaration.
```

Comment:
Actually, one quick thought.  Do we want to explicitly say that for stacks, that the order of `entries` is from element 0 of the stack, then additional elements in ascending order of index?  I know that is the "obvious" thing to do, but couldn't hurt to say explicitly.

Reply:
Sure, I'll update the PR.

Reply:
Done